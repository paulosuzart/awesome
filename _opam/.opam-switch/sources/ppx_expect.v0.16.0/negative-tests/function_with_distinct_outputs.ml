module Expect_test_config = struct
  include Expect_test_config

  let upon_unreleasable_issue = `Warning_for_collector_testing
end

let%expect_test _ =
  let f output =
    print_string output;
    [%expect {| hello world |}]
  in
  f "foo";
  f "bar"
;;
