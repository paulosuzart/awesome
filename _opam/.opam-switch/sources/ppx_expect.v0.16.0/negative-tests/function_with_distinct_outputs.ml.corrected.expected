module Expect_test_config = struct
  include Expect_test_config

  let upon_unreleasable_issue = `Warning_for_collector_testing
end

let%expect_test _ =
  let f output =
    print_string output;
    [%expect {|
      (* expect_test: Collector ran multiple times with different outputs *)
      ======================================================================
      foo
      ======================================================================
      bar |}]
  in
  f "foo";
  f "bar"
;;
